data-raw/R2R_TMH_habitat_inputs/tmh_data_formatting.R

```diff
@@ -0,0 +1,206 @@
+library(tidyverse)
+
+source('data-raw/R2R_TMH_habitat_inputs/tmh_helper_functions.R')
+
+spwn_perc_suitable <- 0.12
+
+floodplain_perc_suitable <- 0.9
+
+rearing_perc_suitable_low <- 0.1
+rearing_perc_suitable_0_1 <- 0.78
+rearing_perc_suitable_1_2 <- 0.78
+rearing_perc_suitable_2_4 <- 0.585
+rearing_perc_suitable_4_8 <- 0.195
+
+run <- 'Fall Run' #'Spring Run' 'Winter Run'
+
+# url <- 'https://docs.google.com/spreadsheets/d/1-3WZAcOzrk5ugZq4CAYK1LRLP0005NmjwpifNQHZ_iA/edit#gid=0'
+
+gradients <- readxl::read_excel('data-raw/R2R_TMH_habitat_inputs/River Length Summary.xlsx', sheet = "Watershed Summary Table") |> 
+  janitor::clean_names() |> 
+  fill(watershed) |> 
+  filter(!is.na(river)) |> 
+  mutate(gradient = ifelse(hqt_boundary == "Valley Lowland (<0.4%)", 0, gradient)) |> 
+  mutate(river_length_ft_grad = as.numeric(length_miles)*5280) |> 
+  select(river = watershed, dam, river_length_ft_grad, gradient) |> 
+  mutate(perc_suitable_rearing = ifelse(gradient < 0, 0.1, 
```

Comment:
please use a case_when not a nested if else

Reply:
or  you could join to a gradient lookup table

Reply:
done